### PR TITLE
Adds PS/2 keyboard support

### PR DESCRIPTION
**Pull request by Elekrisk**
Adds PS/2 keyboard support.

### CODE REVIEW

File: src/printer.rs [hunk 1]
```diff
@@ -9,13 +11,35 @@ const DEFAULT_FONT: [Glyph; 128] = unsafe { core::mem::transmute(*include_bytes!
...
+    [0; 8],
+    [0; 8],
+    [0x00, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x00],
+    [0; 8],
+    [0; 8],
+    [0; 8],
+];
+
+// DON'T COMMIT THIS
+pub static mut PRINTER: Printer = unsafe { Printer::uninitialized() };
```
Comment (brynolv) [inline src/printer.rs]: DON'T COMMIT THIS

Comment (Elekrisk) [inline src/printer.rs]: lol

File: src/printer.rs [hunk 2]
```diff
@@ -39,6 +64,8 @@ impl Printer {
                 }
             }
         }
+        self.line_lengths = SVec::new();
```
Comment (brynolv) [inline src/printer.rs:66]: We could add a `clear()`function to SVec later

**Review by Vikdos — APPROVED** [review]

File: src/ps2_keyboard.rs [hunk 3]
```diff
@@ -1,28 +1,608 @@
...
+            Self::Ö => write!(w, "OE"),
+            _ => write!(w, "{:?}", self)
+        }.unwrap()
+    }
 }
 
 pub enum KeyState {
     Pressed,
-    Held
+    Held,
```
Comment (brynolv) [inline src/ps2_keyboard.rs:569]: `rustfmt`will complain about this, but no problem

File: src/ps2_keyboard.rs [hunk 4]
```diff
@@ -1,28 +1,608 @@
...
 }
 
+#[derive(PartialEq, Eq, Clone, Copy)]
 pub struct Modifiers {
     shift: bool,
     alt: bool,
     altgr: bool,
     ctrl: bool,
-    meta: bool
+    meta: bool,
```
Comment (brynolv) [inline src/ps2_keyboard.rs:578]: Here too

**Review by brynolv — CHANGES_REQUESTED** [review]

File: src/printer.rs (hunk 1 above)
Comment (Elekrisk) [reply src/printer.rs]: lol

File: src/printer.rs (hunk 2 above)
Comment (Elekrisk) [reply src/printer.rs:66]: yup

**Review by brynolv — APPROVED** [review]
Good enough for me for now!

**Pull request merged**